src/features/events/components/ParticipantSummaryCard.tsx

```diff
@@ -0,0 +1,199 @@
+
+                        const intVal = parseInt(val);
+                        if (!isNaN(intVal) && intVal.toString() == val) {
+                          setNewReqParticipants(intVal);
+                        }
+                      }}
+                      onKeyDown={(ev) => {
+                        if (ev.key === 'Enter') {
+                          setAnchorEl(null);
+                          if (newReqParticipants != null) {
+                            model.setReqParticipants(newReqParticipants);
+                          }
+                        } else if (ev.key === 'Escape') {
+                          setAnchorEl(null);
+                          setNewReqParticipants(reqParticipants);
+                        }
+                      }}
+                      value={
+                        newReqParticipants === null ? '' : newReqParticipants
+                      }
+                      variant="outlined"
+                    />
+                  </Box>
+                </Paper>
+              </ClickAwayListener>
+            </Popper>
+          </Box>
+        }
+      >
+        <Box display="flex" justifyContent="space-between">
+          <Box display="flex" flexDirection="column">
+            <Typography color={'secondary'} component="h6" variant="subtitle1">
+              {messages.participantSummaryCard.pending()}
+            </Typography>
+            <Box display="flex">
+              <Typography variant="h4">{signedParticipants}</Typography>
+              {signedParticipants > 0 && (
+                <Button
+                  onClick={() => {
+                    respondents?.map((r) => {
+                      model.addParticipant(r.person.id);
+                    });
+                  }}
+                  startIcon={<Check />}
+                  sx={{
+                    borderColor: 'statusColors.blue',
+                    color: 'statusColors.blue',
+                    marginLeft: 2,
+                  }}
+                  variant="outlined"
+                >
+                  <Msg id={messageIds.participantSummaryCard.bookButton} />
+                </Button>
+              )}
+            </Box>
+          </Box>
+          <Box display="flex" flexDirection="column">
+            <Typography color={'secondary'} component="h6" variant="subtitle1">
+              {messages.participantSummaryCard.booked()}
+            </Typography>
+            <Box display="flex">
+              <Typography variant="h4">{`${remindedParticipants}/${availParticipants}`}</Typography>
+              {remindedParticipants < availParticipants && (
+                <Button
+                  disabled={contactPerson == null}
+                  onClick={() => {
+                    model.sendReminders();
+                  }}
+                  startIcon={<Check />}
+                  sx={{
+                    borderColor: 'statusColors.blue',
+                    color: 'statusColors.blue',
```

Comment:
I didn't know you could access the palette colours like this in sx, that's super convenient 😄

Reply:
This color should not be blue. I know it is blue in the design mockups, but that's because design mockups use another theme (very confusing, I know). The color should just be the default, so these can be taken out.